Px.Utils/PxFile/PxFileSyntaxConf.cs

```diff
@@ -264,13 +263,22 @@ public class EntryValueTypes
 
                 public static EntryValueTypes DefaultEntryValueTypes => new();
 
+                public Dictionary<string, MetaPropertyType> CustomTypeDictionary { get; set; } = [];
```

Comment:
No need to name a dictionary dictionary. Maybe PropertyTypeDefinitions or something like that?

Reply:
Yeah, makes sense